Python/sysmodule.c

```diff
@@ -2861,6 +2861,64 @@ _PySys_InitCore(PyThreadState *tstate, PyObject *sysdict)
     return _PyStatus_ERR("can't initialize sys module");
 }
 
+#ifdef HAVE_FORK
+extern int _PyIO_buffered_at_fork_reinit(PyObject *);
+
+static int
+stdio_at_fork_reinit(_Py_Identifier *key)
+{
+
+    int ret = 0;
+
+    PyObject *isatty = NULL;
+    PyObject *buffer = NULL;
+    PyObject *result = NULL;
+
+    PyObject *stdio = _PySys_GetObjectId(key);
```

Comment:
Looks like this returns a borrowed reference, so there's no need to DECREF it: the code is correct.

Reply:
Cool, thanks. :)

Reply:
nitpick: I suggest the name ``stream``, but stdio works too ;-)